tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -14,6 +14,46 @@
     $(cat {{ edpm_privatekey_path }} | base64 | sed 's/^/        /')
     EOF
 
+- name: set Nova copy shell vars
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.set_fact:
+    nova_copy_shell_vars: |
```

Comment:
FWIW the word "copy" in the variable name i linked to was part of "mariadb_copy" - that role does MariaDB data copying. In this stage of adoption we don't copy anything so it doesn't make all that much sense to me to have "copy" in the var name, but that's a minor thing since the var use is localized and not user facing. Giving +1 overall.